.github/workflows/github-actions-demo.yml

```diff
@@ -0,0 +1,64 @@
+name: GitHub Actions Demo
+run-name: ${{ github.actor }} is testing out GitHub Actions 🚀
+on: [push]
+jobs:
+  Explore-GitHub-Actions:
```

Comment:
The job name should be changed to be a meaningful name.

Reply:
Changed with install-and-test